src/CoreBundle/private/library/classes/TCMSFields/TCMSFieldBlob.class.php

```diff
@@ -68,4 +68,12 @@ public function GetReadOnly()
 
         return '<pre style="white-space:pre">'.$html.'</pre>';
     }
+
+    protected function GetFieldWriterData()
```

Comment:
A blob field is deserialized when loading the data - it could contain any type of data - thus overwriting it's docblock type with `mixed` instead of `string` to reflect that